static/css/techreport/techreport.css

```diff
@@ -224,6 +224,24 @@ nav {
   display: none;
 }
 
+/* Heading links */
+.anchor {
+  color: inherit;
+}
+
+.anchor:is(:hover, :focus-visible) {
+  position: relative;
+}
+
+.anchor:is(:hover, :focus-visible)::after {
+  content: "🔗";
+  display: block;
+  position: absolute;
+  right: -1.3em;
+  top: 0.4em;
```

Comment:
Would `display: inline-block` be better as then don't need to position it absolutely?